### ISSUE TITLE
Torchbench timm_regnet training BF16 & FP16 accuracy regression

### ISSUE THREAD

**Issue by mengfei25** [intel/torch-xpu-ops#1334]
### 🐛 Describe the bug

timm_regnet BF16 gor fail accuracy recently, the last known good is https://github.com/intel/torch-xpu-ops/commit/b6786e31c36b31bb2cc18e2325451a3198832cb8 + torch a7c2d85
```
python benchmarks/dynamo/torchbench.py --accuracy --bfloat16 -d xpu -n10 --training --only timm_regnet --backend=inductor
```
xpu  train timm_regnet                        
E0204 15:16:48.599000 2195001 site-packages/torch/_dynamo/utils.py:2751] RMSE (res-fp64): 0.01081, (ref-fp64): 0.00091 and shape=torch.Size([]). res.dtype: torch.bfloat16, multiplier: 3.000000, tol: 0.001000, use_larger_multiplier_for_smaller_tensor: 0
fail_accuracy

### Versions
torch-xpu-ops: https://github.com/intel/torch-xpu-ops/commit/b6786e31c36b31bb2cc18e2325451a3198832cb8
python: 3.10
TRITON_COMMIT_ID: e98b6fcb8df5b44eb0d0addb6767c573d37ba024
TORCH_COMMIT_ID: 106acf0eec837d93f7373d894c556bec6cb3c265
TORCHBENCH_COMMIT_ID: 373ffb19dc470f4423a3176a4133f8f4b3cdb5bd
TORCHVISION_COMMIT_ID: d23a6e1664d20707c11781299611436e1f0c104f
TORCHAUDIO_COMMIT_ID: 2709b65c9d3c55da40a5436ec4c45c427feb1d2a
DRIVER_VERSION: 803.61
KERNEL_VERSION: 5.15.0-73-generic #80-Ubuntu SMP Mon May 15 15:18:26 UTC 2023
BUNDLE_VERSION: 2025.0.1.20241113
OS_PRETTY_NAME: Ubuntu 22.04.2 LTS
GCC_VERSION: 11

**Comment by jianyizh** [comment]
local test pass

**Comment by jianyizh** [comment]
fail on pytorch 2.6, can pass on current main 1433bc145526949c84acf4ba5eaa1687cc2d72fe

**Comment by weishi-deng** [comment]
The test can pass in the latest pytorch with PR "https://github.com/pytorch/pytorch/pull/126516",

**Comment by jianyizh** [comment]
Fall back fp64 to cpu

**Comment by jianyizh** [comment]
USE #1577 to track
